ci-operator/config/openshift/special-resource-operator/openshift-special-resource-operator-master.yaml

```diff
@@ -7,6 +7,10 @@ base_images:
     name: builder
     namespace: ocp
     tag: rhel-8-golang-1.17-openshift-4.10
+  sro-test-bin:
```

Comment:
Please wait for https://github.com/openshift/release/pull/25922 to merge first and then consume the updated image name.

Reply:
That should not be necessary. The `sro-test-bin` image already exists and is the same image, #25922 just removes its duplicate `test-bin` name.

Reply:
Thanks @petr-muller